disnake/ext/commands/params.py

```diff
@@ -47,13 +47,12 @@
 from disnake.app_commands import Option, OptionChoice
 from disnake.channel import _channel_type_factory
 from disnake.enums import ChannelType, OptionType, try_enum_to_int
+from disnake.interactions import ApplicationCommandInteraction as Interaction
```

Comment:
```suggestion
from disnake.interactions import Interaction
```

Reply:
No this is correct. I used this to shorten the class name. It can stay for now.